### ISSUE TITLE
JAWS not announcing text present in textArea

### ISSUE THREAD

**Issue by swastikgupta21** [FreedomScientific/standards-support#810]
### Summary

JAWS does not announce the text present inside the textArea. It only mentions "contains Text". 
The issue can be reproduced [here](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_textarea)

### Expected result

JAWS should announce the text present inside the text area

### Actual result

It announces "contains text" instead of announcing the text present inside

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

**Comment by JAWS-test** [comment]
The reason is that the text may be too long to output when focusing. The text is output when navigating through the field. So this is not an error. However, it would be a useful feature request that the text is output up to a certain length and a reference to the text is only made for long text or that the first approx. 80 characters are always output and a reference to further characters is made for longer text.
